test/unit/components/reveal.test.js

```diff
@@ -0,0 +1,45 @@
+const expectComponent = require('../component-helper').expectComponent;
+
+describe('Reveal component', () => {
+  it('should render', () => {
+    expectComponent(
+      'reveal',
+      {
+        variant: 'block',
+        summary: 'Summary text',
+        children: [],
+      },
+      `<details class="">
+        <summary data-analytics="summary">
+          <span class="details__summary">
+      Summary text
+    </span>
```

Comment:
Been playing with it. It looks like the white space between tags is ignored. It's only significant when inside tags and that appears to be formatted correctly. So you could do something like:
```js
describe('Reveal component', () => {
  it('should render', () => {
    expectComponent(
      'reveal',
      {
        variant: 'block',
        summary: 'Summary text',
        children: [],
      },
`
<details class="">
  <summary data-analytics="summary">
    <span class="details__summary">
      Summary text
    </span>
  </summary>
  <div>
  </div>
</details>
`
    );
  });
});
```
and it should still pass.